submissions/someusername/aman/py.txt

```diff
@@ -0,0 +1 @@
+print("hello")
```

Comment:
wrong location for file
The file should be in `submissions/aman` and not `submissions/someusername/aman`

Reply:
changed the location of dir.